src/libs/Browser/index.web.ts

```diff
@@ -95,8 +90,9 @@ function openRouteInDesktopApp(shortLivedAuthToken = '', email = '') {
         const iframe = document.createElement('iframe');
         iframe.style.display = 'none';
         document.body.appendChild(iframe);
-        iframe.contentWindow.location.href = expensifyDeeplinkUrl;
-
+        if (iframe.contentWindow) {
+            iframe.contentWindow.location.href = expensifyDeeplinkUrl;
+        }
```

Comment:
Why have you decided to add this condition?

Reply:
Since contentWindow can be `null` we need to check if its exists and in TypeScript, you can't use optional chaining (?.) on the left-hand side of an assignment I needed to make sure that contentWindow exists

Reply:
Okay, makes sense